### PR TITLE
Update links to new Cypher Manual Administration chapter 

### PR DESCRIPTION
**Pull request by JPryce-Aklundh**
Corresponding PR (with more information) in Cypher Manual:
https://github.com/neo4j/docs-cypher/pull/494

### CODE REVIEW

**Comment by neo-technology-commit-status-publisher** [comment]
Looks like you've updated the documentation!

Check out your changes at https://neo4j-docs-operations-635.surge.sh

**Review by renetapopova — APPROVED** [review]
Thanks, @JPryce-Aklundh! All links look good to me. I haven't checked if there are any links that you've missed.

**Review by davidoliverSP2 — APPROVED** [review]
This list looks complete, thanks @JPryce-Aklundh

File: modules/ROOT/pages/clustering/servers.adoc [hunk 1]
```diff
@@ -251,7 +251,7 @@ neo4j@neo4j> ALTER SERVER '25a7efc7-d063-44b8-bdee-f23357f89f01' SET OPTIONS {mo
 Altering servers may cause databases to be moved, and should be performed with care.
 For example, if the server `25a7efc7-d063-44b8-bdee-f23357f89f01` hosts database `foo` in primary mode when the above command is executed, then another server must begin hosting `foo` in primary mode.
 Likewise, if `ALTER SERVER '25a7efc7-d063-44b8-bdee-f23357f89f01' SET OPTIONS {allowedDatabases:['bar','baz']};` is executed, then `foo` is forced to move.
-For a description of all the server options (e.g., server tags) that can be altered via the `ALTER SERVER` command, see the xref:link:{neo4j-docs-base-uri}/cypher-manual/{page-version}/access-control/server-management/#server-management-alter-server[Cypher Manual -> Modifying servers].
+For a description of all the server options (e.g., server tags) that can be altered via the `ALTER SERVER` command, see the xref:link:{neo4j-docs-base-uri}/cypher-manual/{page-version}/administration/access-control/server-management/#server-management-alter-server[Cypher Manual -> Modifying servers].
```
Comment (Hunterness) [inline modules/ROOT/pages/clustering/servers.adoc:254]: didn't the server command section move out from access-control?

File: modules/ROOT/pages/authentication-authorization/sso-integration.adoc [hunk 2]
```diff
@@ -181,7 +181,7 @@ dbms.security.oidc.mysso.authorization.group_to_role_mapping=\
 <3> Mapping of two identity provider groups to a Neo4j built-in role.
 <4> Mapping of an identity provider group to a custom-defined role.
 Custom-defined roles, such as `rolename`, must be explicitly created using the `CREATE ROLE rolename` command before they can be used to grant privileges.
-See link:{neo4j-docs-base-uri}/cypher-manual/{page-version}/access-control/manage-users[the Cypher Manual -> Creating roles].
+See link:{neo4j-docs-base-uri}/cypher-manual/{page-version}/administration/access-control/manage-users[the Cypher Manual -> Creating roles].
```
Comment (Hunterness) [inline modules/ROOT/pages/authentication-authorization/sso-integration.adoc:184]: why are we linking to manage users when talking about creating roles?

File: modules/ROOT/pages/authentication-authorization/ldap-integration.adoc [hunk 3]
```diff
@@ -123,7 +123,7 @@ dbms.security.ldap.authorization.group_to_role_mapping=\
 <3> Mapping of two LDAP groups to a Neo4j built-in role.
 <4> Mapping of an LDAP group to a custom-defined role.
 Custom-defined roles, such as `rolename`, must be explicitly created using the `CREATE ROLE rolename` command before they can be used to grant privileges.
-See link:{neo4j-docs-base-uri}/cypher-manual/{page-version}/access-control/manage-users[the Cypher Manual -> Creating roles].
+See link:{neo4j-docs-base-uri}/cypher-manual/{page-version}/administration/access-control/manage-users[the Cypher Manual -> Creating roles].
```
Comment (Hunterness) [inline modules/ROOT/pages/authentication-authorization/ldap-integration.adoc:126]: here as well, linking to users when talking about roles

File: modules/ROOT/pages/authentication-authorization/introduction.adoc [hunk 4]
```diff
@@ -25,7 +25,7 @@ For more information, see xref:configuration/password-and-user-recovery.adoc[Pas
 When triggered, Neo4j logs an error containing a timestamp and the message `failed to log in: too many failed attempts` in the _security.log_.
 ====
 +
-The Cypher commands to manage users, roles, and permissions are described in detail in link:{neo4j-docs-base-uri}/cypher-manual/{page-version}/access-control/[Cypher Manual -> Administration].
+The Cypher commands to manage users, roles, and permissions are described in detail in link:{neo4j-docs-base-uri}/cypher-manual/{page-version}/administration/access-control/[Cypher Manual -> Administration].
```
Comment (Hunterness) [inline modules/ROOT/pages/authentication-authorization/introduction.adoc:28]:
```suggestion
The Cypher commands to manage users, roles, and permissions are described in detail in link:{neo4j-docs-base-uri}/cypher-manual/{page-version}/administration/access-control/[Cypher Manual -> Access control].
```

**Review by Hunterness — APPROVED** [review]

**Pull request merged**